core/src/i18n/en.yaml

```
@@ -31,21 +31,26 @@ common:
   buttons:
     cancel: Cancel
     delete: Delete
+    edit-yaml: Edit YAML
+  created: Created
   delete-dialog:
     delete-confirm: Don't ask this again
     message: Are you sure you want to remove {{type}} {{name}}?
     title: Remove {{name}}
+  labels: Labels
+  name: Name
+  status: Status
   tooltips:
-    download: Download
-    search: Search
-    save: Save
     copy-to-clipboard: Copy to clipboard
+    download: Download
     duplicate-key: Duplicate key
-    k8s-name-input: Name must contain lower case alphanumeric characters, can contain '-' (like 'my-name1').
-    key-value: key=value, must start and end with alphanumeric character, can contain '-', '_' or '.'
-    read-file: Read file content as single value, with file name as a key.
+    k8s-name-input: Name must contain lowercase alphanumeric characters, can also contain '-' (like 'my-name1').
```

```suggestion
    k8s-name-input: Name must consist of lowercase alphanumeric characters, can also contain '-' (like 'my-name1').
```

See [this comment](https://github.com/kyma-project/busola/pull/323#discussion_r685826802).